_data/monthly-calls.yml

```
@@ -1,16 +1,42 @@
 videos:
+  - title: Building with USWDS tools
+    subtitle: Learn how to build new components that look and feel consistent with existing components.
+    description: |
+     Our USWDS team did a great live [tutorial on customizing and extending USWDS](https://github.com/uswds/uswds-tutorial). We covered working with settings, applying tokens, handling style conflicts, and ensuring good color contrast.
```

```suggestion
     Our USWDS team did a live [tutorial on customizing and extending USWDS](https://github.com/uswds/uswds-tutorial). We covered working with settings, applying tokens, handling style conflicts, and ensuring good color contrast.
```

Removed `great` not because it wasn't but to remove too much self-editorializing. It'll be up to the viewer to determine how great it was!